src/core/utils-new.js

```diff
@@ -188,6 +188,45 @@ export default class UtilsNew {
         }
     }
 
+    static draggableModal(self, modalElm) {
+        let mousePosition;
+        let offset = [0, 0];
+        let isDown = false;
+        const modalDialog = modalElm.querySelector(".modal-dialog");
+        const modalHeader = modalElm.querySelector(".modal-header");
+        if (modalDialog) {
+            modalDialog.style.margin = "0";
+            modalDialog.style.left = (window.innerWidth * 0.30) + "px";
+            modalDialog.style.top = (window.innerHeight * 0.05) + "px";
+        }
+
+        modalHeader.addEventListener("mouseover", e => e.currentTarget.style.cursor = "move");
```

Comment:
@Rodielm Sorry, but this is completely useless, as you can just set the cursor style without adding any event listener:

```js
modalHeader.style.cursor = "move";
```

The `cursor` style property is only applied when the pointer is over the element: https://developer.mozilla.org/en-US/docs/Web/CSS/cursor